experimenter/experimenter/nimbus_ui_new/tests/test_forms.py

```diff
@@ -0,0 +1,73 @@
+from unittest import mock
+
+from django.test import TestCase
+
+from experimenter.experiments.models import NimbusExperiment
+from experimenter.experiments.tests.factories import NimbusExperimentFactory
+from experimenter.nimbus_ui_new.constants import NimbusUIConstants
+from experimenter.nimbus_ui_new.forms import NimbusExperimentCreateForm
+from experimenter.openidc.tests.factories import UserFactory
+
+
+class TestNimbusExperimentCreateForm(TestCase):
+    def setUp(self):
+        self.user = UserFactory.create(email="dev@example.com")
+        self.request = mock.Mock()
+        self.request.user = self.user
```

Comment:
This should use RequestFactory instead of Mock:
https://docs.djangoproject.com/en/5.0/topics/testing/advanced/#the-request-factory